<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Prasath17" failures="1" tests="4" name="prasath.Testcase.BasicflowTest" time="36.257" errors="0" timestamp="15 Jul 2024 10:34:36 GMT" skipped="0">
  <testcase classname="prasath.Testcase.BasicflowTest" name="login" time="3.032"/>
  <testcase classname="prasath.Testcase.BasicflowTest" name="addProductToCart" time="4.448"/>
  <testcase classname="prasath.Testcase.BasicflowTest" name="verifyProductInCart" time="15.568"/>
  <testcase classname="prasath.Testcase.BasicflowTest" name="completePurchase" time="13.209">
    <failure message="Desired product not found: ZARA COAT 3 expected object to not be null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Desired product not found: ZARA COAT 3 expected object to not be null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.assertNotNull(Assert.java:686)
at prasath.Testcase.BasicflowTest.completePurchase(BasicflowTest.java:82)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- completePurchase -->
</testsuite> <!-- prasath.Testcase.BasicflowTest -->
